tests/unit/amo/components/TestRatingManager.js

```diff
@@ -581,6 +577,39 @@ describe(__filename, () => {
         "Nevermind, I don't want to edit my review",
       );
     });
+
+    it('shows AddonReviewCard with a saved review', () => {
+      const location = createFakeLocation({ pathname: '/the/detail/page' });
+      const review = {
+        ...fakeReview,
+        body: 'This is hands down the best ad blocker',
+      };
+      const root = renderInline({
+        location,
+        store: createStoreWithLatestReview({ review }),
+      });
+
+      const reviewCard = root.find(AddonReviewCard);
+      expect(reviewCard).toHaveProp('review', createInternalReview(review));
+      expect(reviewCard).toHaveProp('location', location);
+    });
+
+    it('configures AddonReviewCard to show a write review button for ratings', () => {
+      const review = { ...fakeReview, body: undefined };
+      const root = renderInline({
+        store: createStoreWithLatestReview({ review }),
+      });
+
+      const reviewCard = root.find(AddonReviewCard);
+      expect(reviewCard).toHaveProp('bodyFallback');
+
+      const bodyFallback = shallow(reviewCard.prop('bodyFallback'));
+      bodyFallback.find(Button).simulate('click', createFakeEvent());
```

Comment:
This is kind of two tests in one, which is fine by me, but I feel like maybe I've been called out on that in the past.

1. Displays a button to write a review for a rating.
2. Clicking the button opens the edit form.

Reply:
It makes sense to me as one test. It's testing a piece of logic; a *write a review* button that you can click on. I think the name of the test spec was just wrong so I updated it.